etc/scripts/julia_wrapper.jl

```diff
@@ -19,21 +19,21 @@ Options:
 
 using InteractiveUtils
 
-if length(ARGS) < 2
-	println(doc)
-	exit(1)
+if first(ARGS) == "--"
```

Comment:
@aw32 I am not sure I understand the point of the `--`.

Reply:
Usually, this is a hint to a program to stop parsing option-arguments and interpret the following arguments beginning with `--` as not-option-arguments.
For example, `grep --color -- --count` tells grep to interpret `--count` as PATTERN.

In this case, AFAIK, the `--` was added to the commandline of Julia so the UI arguments are passed to the wrapper script instead of being interpreted by Julia.
Else you could pass arbitrary arguments to the Julia call using the ui.
However, because the wrapperscript name already ends the optional arguments of Julia, the `--` may be without a function.

Reply:
Yeah in Julia it's position dependent and the args after the script name are the ones that matter.